src/gurt/telemetry.c

```diff
@@ -3162,35 +3162,33 @@ list_children(struct d_tm_context *ctx, struct d_tm_nodeList_t **head,
  *					from this node.
  * \param[in,out]	num_dirs	Optional pointer to storage for the
  *					number of directories found.
- * \param[in]		max_depth	The maximum depth of the listing, including
- *					the root node.
+ * \param[in]		max_depth	The maximum depth below the root of
+ *					the listing.
  *
  * \return		DER_SUCCESS		Success
  *			-DER_NOMEM		Out of global heap
  *			-DER_INVAL		Invalid pointer for \a head or
  *						\a node
  */
 int
-d_tm_list_subdirs(struct d_tm_context *ctx, struct d_tm_nodeList_t **head,
+d_tm_list_subdirs(struct d_tm_context *ctx, struct d_tm_nodeList_t **list,
 		  struct d_tm_node_t *node, uint64_t *num_dirs, int max_depth)
 {
 	int			 rc = DER_SUCCESS;
 	uint64_t		 dir_count = 0;
-	struct d_tm_nodeList_t	*cur_head = NULL;
+	struct d_tm_nodeList_t	*head = NULL;
```

Comment:
A nit, but the more I look at this, the less I think this variable should be called head, since it's only used to iterate through the list. Maybe "cur" or something like that?

Reply:
I agree that these could be clearer. Adjusted to follow convention which should make it easier to comprehend.